playbooks/tasks/create_server_groups.yaml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Create Server Groups for nodes
+  block:
+    - name: Ensure pip is installed
+      ansible.builtin.package:
+        name: python3-pip
+        state: present
+      become: true
+    - name: Install openstack cli
+      ansible.builtin.pip:
+        name:
+        - "python-openstackclient=={{ openstackclient_version }}"
+        extra_args: --user
+    - name: Create Server Group for contole-plane nodes
+      ansible.builtin.command: "openstack --os-compute-api-version 2.15 server group create --policy anti-affinity -f value -c id test-cluster-controller"
+      register: srvgrp_controller
+      environment:
+        OS_CLOUD: "{{ cloud_name }}"
+      changed_when: true
+    - name: Create Server Group for worker nodes
+      ansible.builtin.command: "openstack --os-compute-api-version 2.15 server group create --policy soft-anti-affinity -f value -c id test-cluster-worker"
```

Comment:
We probably do not need `--os-compute-api-version 2.15`

Reply:
Yes, I think it was only necessary for `openstack CLI` version like `5.x` or older, that I had installed locally. I will remove it.